app/src/main/java/com/codefororlando/petadoption/presenter/list/AddAnimalsAction.java

```diff
@@ -0,0 +1,25 @@
+package com.codefororlando.petadoption.presenter.list;
+
+import com.codefororlando.petadoption.data.model.Animal;
+import com.codefororlando.petadoption.recyclerview.AAnimalListAdapter;
+
+import java.util.List;
+
+import io.reactivex.annotations.NonNull;
+import io.reactivex.functions.Consumer;
+
+class AddAnimalsAction implements Consumer<List<Animal>> {
+
+    private final AAnimalListAdapter animalListAdapter;
```

Comment:
Is AA a type-o?

Reply:
The first A is for "abstract". Should probably name it better outside of this PR.